superset/views/core.py

```diff
@@ -1118,29 +1114,19 @@ def get_samples(self, viz_obj):
             'data': viz_obj.get_samples(),
         })
 
+    @handle_superset_exception
```

Comment:
Just rebased and trying to fix stacktraces, and I think applying the decorator here is wrong since this method is not a view